concordium-sdk/src/main/java/com/concordium/sdk/transactions/TransferWithMemo.java

```diff
@@ -38,6 +42,12 @@ byte[] getBytes() {
         buffer.put(getAmount().getValue().getBytes());
         return buffer.array();
     }
+    public static TransferWithMemo fromBytes(ByteBuffer source) {
```

Comment:
As my other comments, does this need to be exposed? 
Otherwise I suggest
```suggestion

    static TransferWithMemo fromBytes(ByteBuffer source) {
```

Reply:
I don't understand the reasons for hiding these methods. How is anybody meant to parse things otherwise?